x/ccv/provider/client/handler.go

```diff
@@ -0,0 +1,24 @@
+package client
```

Comment:
Why is this file and and the _test in provider/client? In the sdk they are in module directories. So it would be in provider/ here.